python/neuroglancer/server.py

```diff
@@ -29,6 +29,7 @@
 import tornado.web
 
 import sockjs.tornado
+from sockjs.tornado.util import asynchronous
```

Comment:
Can you change this to conditionally import either this definition or the one from tornado.web if available, so that this continues to work with old versions of tornado and sockjs.tornado as well?

Reply:
I assume that means you want me to drop the dependency constraint on sockjs-tornado.   Will there be problems with someone using the newest sockjs, but an older tornado (or vice versa)?

Reply:
I think it is alright --- pip doesn't really strictly enforce dependency constraints anyway.